js/main.js

```diff
@@ -25,7 +25,9 @@ Site = {
       }
 
       $(window).bind('resizeEnd', function() {
-        _this.Journal.masonryLayout();
+        if ($('body').hasClass('blog')) {
```

Comment:
if this bind is only needed for blog section better to put the conditional outside the bind. Then it only runs once and the bind doesnt happen when not needed